fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -440,6 +440,14 @@ public UserVariable getUserVariable(String variable) {
         return userVariables.get(variable);
     }
 
+    public void setUserVariables(HashMap<String, UserVariable> userVariables) {
```

Comment:
I'm wondering if the user variable is case-sensitive or case-insensitive, and how we guarantee it works.

Reply:
It's case-sensitive.  We always use a case-sensitive map to store them.